src/api/rest/server/mcir/sshkey.go

```diff
@@ -136,3 +136,26 @@ func RestDelAllSshKey(c echo.Context) error {
 	// Obsolete function. This is just for Swagger.
 	return nil
 }
+
+func RestTestSetSshKeyInUseCount(c echo.Context) error {
+
+	nsId := c.Param("nsId")
+	sshKeyId := c.Param("sshKeyId")
+
+	inUseCount, err := mcir.SetInUseCount(nsId, "sshKey", sshKeyId, "+1")
```

Comment:
```suggestion
	inUseCount, err := mcir.SetInUseCount(nsId, resTypeSshKey, sshKeyId, "+1")
```

Static text 사용을 줄이기 위해서, 
const resTypeSshKey string = "sshKey"
와 유사한 방법으로 상수로 정의해서 사용하는 것은 어떨까요?

해당 사항은 해당 PR에만 관련된 사항이 아니므로 ~ 필요시 향후에 처리해도 될 것 같습니다.

```
	if resourceType == "image" ||
		resourceType == "sshKey" ||
		resourceType == "spec" ||
		resourceType == "vNet" ||
		//resourceType == "subnet" ||
		//resourceType == "publicIp" ||
		//resourceType == "vNic" ||
		resourceType == "securityGroup" {
		// continue
	} else {
		errString := "Cannot list " + resourceType + "s."
		err := fmt.Errorf(errString)
		return nil, err
	}
```

Reply:
좋아 보입니다.
다음 PR 때 반영하겠습니다.